.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+Before submitting your PR, check that you've:
+
+- [ ] Documented new APIs with [Haddock markup](https://www.haskell.org/haddock/doc/html/index.html)
+- [ ] Added [`@since` declarations](http://haskell-haddock.readthedocs.io/en/latest/markup.html#since) to the Haddock
```

Comment:
I *really* want this. It's very nice to have `@since` annotations on code to establish lower bounds.

Reply:
Contributors might not know what the next version number is going to be.

One solution is to preemptively create a changelog entry of X.Y.(Z+1).0 where X.Y.Z.P is the latest released version.

This also raises the questions of whether and how to manage incrementing the 4-th number, for releases without API additions. And also how to fit breaking changes in this workflow.